tools/getkeys/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    "extends": [
+        require.resolve("@fluidframework/eslint-config-fluid")
+    ],
+    "parserOptions": {
+        project: ["./tsconfig.json"],
+    },
+    "rules": {
+        "@typescript-eslint/ban-ts-comment": "off",
+        "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/no-unsafe-return": "off",
+        "@typescript-eslint/restrict-plus-operands": "off",
+        "@typescript-eslint/strict-boolean-expressions": "off",
+        "@typescript-eslint/no-unsafe-assignment": "off",
+        "@typescript-eslint/no-unsafe-argument": "off",
+        "@typescript-eslint/no-unsafe-member-access": "off",
+        "@typescript-eslint/no-unsafe-call": "off",
+        "promise/param-names": "off"
+    }
```

Comment:
Have we tried removing these rule overrides from here, and see which ones are actually broken in getkeys? I suspect at least some of these are not required in this package.

Reply:
Yes, I have tried with these rules then we will need to remove all 'any' var. These are minimum required for getkey package.

Reply:
Please don't use project-level disables except for maybe the no-unsafe-\* ones. But some of those are disabled already in the shared config, so you shouldn't need to disable them here:

https://github.com/microsoft/FluidFramework/blob/f3c71e6ecef92db6a48b8dd77d3db2a7fbd3537d/common/build/eslint-config-fluid/printed-configs/minimal.json#L228-L236

Reply:
I see. Then seems like I didn't include this extend config. Troubleshooting...

Reply:
Seems like I should include `@fluidframework/eslint-config-fluid/minimal` in eslintrc extends. Thanks for pointing out. Good catch!